Flow.Launcher.Plugin/Query.cs

```diff
@@ -16,6 +16,9 @@ public Query(string rawQuery, string search, string[] terms, string[] searchTerm
         {
             Search = search;
             RawQuery = rawQuery;
+#pragma warning disable 618 Legacy Support
```

Comment:
what happens if you dont disable the warning?

Reply:
Just it will have a warning here when editing with ide.

Reply:
When get a chance could you screenshot the warning? I think the warning would be fine right since this is will let devs know this is going to be removed at some point.

Reply:
ok